README.md

```diff
@@ -70,6 +70,10 @@ Got issues with what you find here? You can [create an issue on Github](https://
 
 If you'd like to take a crack at making some changes, please refer to [our contributing guide](.github/CONTRIBUTING.md). 
 
+# Local development
+
+You can run `yarn --cwd /path/to/zcli dev` in a development environment without transpiling ZCLI into JavaScript.
+
```

Comment:
could you leave an example here? I usually use something like `yarn dev apps:server ./zcli-demo-4`

Reply:
I added an example in ee8dd0d. Does it work for you?